superdesk/publish/subscribers_schedule.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from superdesk import get_resource_service
+from superdesk.celery_app import celery
+from superdesk.utc import utcnow
+
+logger = logging.getLogger(__name__)
+
+
+@celery.task(soft_time_limit=30)
+def update_subscriber_activation_states():
+    """Task to check scheduled subscribers and activate/deactivate them.
+
+    Runs every minute. If a subscriber has a `schedule` field defined,
+    this will check if the current time is within the start and end range
+    and update `is_active` accordingly.
+    """
+    now = utcnow()
+    service = get_resource_service("subscribers")
+
+    try:
+        # Query for subscribers that have a schedule defined
+        lookup = {
+            "$and": [
```

Comment:
would be good to only get those that need some changes, so the query could be like:
```
today = get_local_today().date().isoformat()
{"$or": [
    {"schedule.startDate": today},
    {"schedule.endDate": today},
]}
```